### PR TITLE
[CWS][SEC-4878] add agent version support to macros

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

This PR add the agent version constraint for macros. It uses the same syntax as for the rules.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: releasenotes/notes/cws-add-macro-agent-version-support-317539fb49fa7584.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    CWS now supports for agent version constraints for macro.
```
Comment (kayayarai) [inline releasenotes/notes/cws-add-macro-agent-version-support-317539fb49fa7584.yaml:4]:
```suggestion
    Cloud Workload Security now has Agent version constraints for macros.
```
I think? Does a user need more explanation to understand what this means? I don't really understand just from this description.

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: releasenotes/notes/cws-add-macro-agent-version-support-317539fb49fa7584.yaml (hunk 1 above)
Comment (safchain) [reply releasenotes/notes/cws-add-macro-agent-version-support-317539fb49fa7584.yaml:4]: Maybe : Cloud Workload Security now has Agent version constraints for Macros in SECL expressions. ?

**Review by paulcacheux — APPROVED** [review]

**Comment by safchain** [comment]
@kayayarai is the new version OK for you ?

**Review by kayayarai — APPROVED** [review]
Yeah! Like it.

**Pull request merged**